app/Http/Controllers/CommentController.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Comment;
+use App\Http\Requests\CommentFormRequest;
+use App\Http\Resources\CommentResource;
+use App\Http\Resources\ReplyCommentResource;
+use App\Image;
+use App\Traits\CommentTrait;
+use App\Traits\PostTrait;
+use Illuminate\Http\Request;
+use App\Traits\ImageTrait;
+use Exception;
+use Illuminate\Support\Facades\DB;
+
+class CommentController extends Controller
+{
+    use PostTrait, ImageTrait, CommentTrait;
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(CommentFormRequest $request)
+    {
+
+        $comment = new Comment();
+        $comment->content = $request->content;
+        $comment->user_id = $request->user()->id;
+
+        DB::beginTransaction();
```

Comment:
check cách sử dụng DB::transaction, sau đó cho anh so sánh giữa cách này với cách hiện tại hỉ

Reply:
DB::transaction tiện hơn nhé anh, cách e code đang dùng tay, hôm trước em thử mà nó báo luôn vì e thiếu this trong function DB::transaction  :D, em mới sửa xong.